components/layouts/Footer.jsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
+import Wrapper from '../wrappers/Wrapper';
+import Button from '../../components/ui/Button'
+import Link from 'next/link';
 
 function Footer(props) {
     return (
-        <div>
-            <h1>Footer</h1>
-        </div>
+        <Wrapper>
+           <div className='flex flex-col sm:flex-row justify-between gap-y-12 sm:gap-y-0 text-xl md:text-2xl font-normal leading-8 tracking-thiner md:tracking-thin'>
+            <div>
+                <ul>
```

Comment:
According to the figma design, this is a paragraph you'll have to give it just a width that it follows, don't make it a list and also remove the break, let it break it's self depending on a width or anything else!